.setup/bin/populate_office_hours_queue.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+
+import argparse
+from sqlalchemy import create_engine, Table, MetaData, select, update, insert
+import random
+import os
+from datetime import datetime, timedelta
+
+DB_HOST = "localhost"
+DB_PORT = 5432
+DB_USER = "submitty_dbuser"
+DB_PASS = "submitty_dbuser"
+
+def main():
+    # get today's date to determine date and time
+    today = datetime.today()
```

Comment:
There's a similar function in the python submitty utils lib that gets either the summer or fall term.

```python
from submitty_utils import dateutils

_ = dateutils.get_current_semester()
```

The function is under `python_submitty_utils/submitty_utils/dateutils.py`, it doesn't handle `u` for summer terms but I think that's ok since this is just sample data